apps/bare-expo/app.json

```diff
@@ -44,6 +44,11 @@
     "runtimeVersion": {
       "policy": "appVersion"
     },
+    "updates": {
+      "checkAutomatically": "NEVER",
+      // Workaround to use expo/expo URL since we don't have an EAS update project for bare-expo yet.
```

Comment:
I think `//` comments are invalid json?

Reply:
we support json5 in app.json and eas.json 😅